### ISSUE TITLE
Example of using a custom handler?

### ISSUE THREAD

**Issue by tademianhess** [serverless-nextjs/serverless-next.js#918]
Is there documentation or example code for specifying a custom handler with the serverless-next component (is that even still an option)? 

I see this note for the "handler" property on https://www.serverless.com/plugins/serverless-nextjs-plugin:
"When assigned a value, overrides the default function handler to allow for configuration. Copies handler.js in route into the Lambda folders. Your handler MUST still call the default-handler afterwards or your function won't work with Next.JS"

Is that still a valid option that can be set for the serverless-next component?

**Comment by dphang** [comment]
I think you can refer to the PR for more details: https://github.com/serverless-nextjs/serverless-next.js/pull/649. I haven't used this myself but hopefully @emhagman might be able to help in case you are stuck :)

**Comment by tademianhess** [comment]
@dphang Excellent! Thanks

**Comment by ekasprzyk** [comment]
I tried using this feature with [Lumigo](https://lumigo.io/) and since this is the top link when searching for this feature, I figured I'd update this closed issue.  

The pull request and the documentation around sentry gloss over a lot, so I've added a guide (in the PR) for how I managed to deploy a library to the lambda and use it from the service in the handler, something that seems to be missing from the docs.  

Hope it's useful and maybe someone will figure out a more elegant way.
